docs/login.puml

```diff
@@ -0,0 +1,35 @@
+@startuml login
+
+(*) if "Started from QR code scanner?" then
+-[#red]->[Yes] "Validate server details"
+else
+    ->[No] if "Has account?" then
+    -->[Yes] (*)
+    else
+    -->[No] "Login page"
+    endif
+endif
+
+if "Selected login method?" then
+-[#blue]->[Login flow] "Validate server details"
+-[#blue]-> "Open login flow page"
+-[#blue]-> "Initiate login flow"
+-[#blue]-> "Open web browser"
+-[#blue]-> "Wait for login flow result"
+-[#blue]-> "Validate login details"
```

Comment:
This one is obsolete, as we can be sure that the credentials returned by the login flow are correct (nevertheless we need to fetch the username, so we might as well leave it in)

Reply:
@Leptopoda Should we leave it or not? The logic would be exactly the same and we should also display some UI